src/d3d12/d3d12_acceleration_structure..cpp

```diff
@@ -329,7 +330,8 @@ namespace wr::d3d12
 		else if (GetRaytracingType(device) == RaytracingType::FALLBACK)
 		{
 			// Set the descriptor heaps to be used during acceleration structure build for the Fallback Layer.
-			d3d12::BindDescriptorHeaps(cmd_list, { desc_heap }, 0, true);//TODO: note this non frame idx
+			//d3d12::BindDescriptorHeaps(cmd_list, { desc_heap }, 0, true);//TODO: note this non frame idx
```

Comment:
Don't comment out code. Remove it if its unnecessary. We are using version control so we can always revert.